src/http/middleware/Auth.js

```diff
@@ -0,0 +1,54 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import db from '../../db';
+
+dotenv.config();
+
+// import/prefer-default-export
+const Auth = async (req, res, next) => {
```

Comment:
Any reason you decided to capitalize `Auth`?

Reply:
fixed